From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 19 Nov 2015 16:53:42 +0100
Subject: [PATCH] HACK: include missing.h in btrfs-ctree.h for BTRFS_UUID_SIZE

This is needed for older Kernel headers without btrfs.h

Not for upstream, but needed for host-systemd.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 src/basic/btrfs-ctree.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/btrfs-ctree.h b/src/basic/btrfs-ctree.h
index 66bdf9736ef5..e9edc8eb5445 100644
--- a/src/basic/btrfs-ctree.h
+++ b/src/basic/btrfs-ctree.h
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "missing.h"
 #include "macro.h"
 #include "sparse-endian.h"
 
